keyboards/viktus/tx_roundup_pad/readme.md

```diff
@@ -0,0 +1,27 @@
+# TX Roundup Pad
+
+![tx_roundup_pad]()
```

Comment:
Image link here?

Reply:
There isn't a publicly hosted one at this time, but should be in the future.